lib/fps_counter.dart

```diff
@@ -36,4 +36,4 @@ mixin FPSCounter on Game {
     final double averageDt = dtSum / average;
     return 1 / averageDt;
   }
-}
\ No newline at end of file
+}
```

Comment:
oh our imperfect linter!

Reply:
I saw this review on the discord and thought I checked by.

I noticed that the `flutter format` command is used for linting but technically that is more of an analyze kind of job. I would recommend something like `dartdoc --no-auto-include-dependencies --quiet`.

This has caught my linting on my own flutter plugins quite often. Use that with a `dartdoc_options.yaml` and a `analyze_options.yaml` and you have something that follows both pub.dev rules and customization from your own part.

Links to two examples that I use of those files:
- [analyze_options.yaml](https://gitlab.com/twilio-flutter/programmable-video/-/blob/master/programmable_video/analysis_options.yaml)
- [dartdoc_options.yaml](https://gitlab.com/twilio-flutter/programmable-video/-/blob/master/programmable_video/dartdoc_options.yaml)

Do with this information what you wanted :-) just couldnt help myself

Reply:
oh man, I love this! I think we should *definitely* do this. @wolfenrain are you interested in making a PR? otherwise I will hehehe. great info!